CHANGES.rst

```diff
@@ -15,6 +15,7 @@ Version 3.1.0 (2019-XX-XX)
     + ``io.eager.update_dataset_from_dataframes``
 - fix ``_apply_partition_key_predicates`` ``FutureWarning``
 - serialize :class:`~kartothek.core.index.ExplicitSecondaryIndex` to parquet
+- remove pandas schema metadata during empty dataframe creation for `pyarrow>=0.13`
```

Comment:
Try to express the change in behavior / the fixed issue in the changelog. the changelog is there to help the user understand what's going on without knowing the implementation. Without knowing the issue even I wouldn't know what the implications of removing this metadata would be.